crates/bevy_transform/src/components/global_transform.rs

```diff
@@ -211,6 +211,28 @@ impl GlobalTransform {
         self.0.translation
     }
 
+    /// Get the rotation as a [`Quat`].
+    ///
+    /// The transform is expected to be non-degenerate and without shearing, or the output will be invalid.
+    /// 
+    /// It is calculated using `to_scale_rotation_translation`, meaning that you
+    /// shoud probably use it directly if you also need translation or scale.
+    #[inline]
+    pub fn just_rotation(&self) -> Quat {
+        self.to_scale_rotation_translation().1
+    }
+
+    /// Get the scale as a [`Vec3`].
```

Comment:
This also needs the caveats about being shear-free, otherwise this is not well-defined. The behavior in the presence of a negative determinant should maybe be documented as well.